compiler/rustc_hir/src/def.rs

```diff
@@ -275,17 +379,23 @@ impl PartialRes {
     }
 }
 
-/// Different kinds of symbols don't influence each other.
-///
-/// Therefore, they have a separate universe (namespace).
+/// Different kinds of symbols can coexist even if they share the same textual name.
+/// Therefore, they each have a separate universe (known as a "namespace").
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
 pub enum Namespace {
+    /// The type namespace includes `struct`s, `enum`s, `union`s, `trait`s, and `mod`s
+    /// (and, by extension, crates).
```

Comment:
apparently also tool attribute modules if the comment for that res is correct. Not sure how sensible it is to exhaustively enumerate everything that's possible here.

Might be good to put an "e.g." in front here